src/ArrayOfSetsView.hpp

```diff
@@ -300,7 +300,7 @@ class ArrayOfSetsView : protected ArrayOfArraysView< T, INDEX_TYPE, std::is_cons
     ARRAYOFARRAYS_CHECK_BOUNDS( i );
 
     INDEX_TYPE const setSize = sizeOfSet( i );
-    T const * const setValues = (*this)[i];
+    T * const setValues = getSetValues( i );
```

Comment:
How did this work before when `setValues` was a pointer to const?

Reply:
The `setValues` is used to determine the number of insertions that take place, then the `incrementSize` call back returns the pointer that is used to do the insertions.